scripts/removejujuservices.bash

```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+# Copyright 2024 Canonical Ltd.
+# Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
Do we need this?

Reply:
Fair, I've changed my automated copyright thing to just do Go files